staging/src/k8s.io/legacy-cloud-providers/gce/gce_instances.go

```diff
@@ -458,37 +489,37 @@ func (g *Cloud) getInstancesByNames(names []string) ([]*gceInstance, error) {
 				continue
 			}
 			if found[inst.Name] != nil {
-				klog.Errorf("Instance name %q was duplicated (in zone %q and %q)", inst.Name, zone, found[inst.Name].Zone)
+				klog.Errorf("Instance name %q was duplicated (in zone %q and %q)", inst.Name, zone, found[inst.Name].instance.Zone)
```

Comment:
guard against nil pointer for `instance`?

Reply:
I don't think its possible for instances to include any nils.

Reply:
not in the current iteration of the code, but code changes over time :)